lightly/api/api_workflow_compute_worker.py

```diff
@@ -85,11 +87,15 @@ def create_compute_worker_config(
             The id of the created config.
 
         """
+        if isinstance(selection_config, dict):
+            selection = _selection_config_from_dict(cfg=selection_config)
+        else:
+            selection = selection_config
```

Comment:
I would really consider to not support both versions, just choose one.

If we go with `SelectionConfig` let's make `_selection_config_from_dict` a public function and use it in examples.

Reply:
I would allow both for the moment.